data/json/items/ranged.json

```diff
@@ -135,7 +135,8 @@
     "durability": 8,
     "burst": 5,
     "clip_size": 100,
-    "reload": 350
+    "reload": 350,
+    "valid_mod_locations": [[ "accessories", 4 ],[ "barrel", 1 ],[ "grip", 1 ],[ "muzzle", 1 ],[ "rail", 1 ],[ "sights", 1 ],[ "stock", 1 ],[ "underbarrel", 1 ]]
```

Comment:
Well, I guess the BB gun needs checked up then, since it shouldn't have Picatinny rails, a pistol grip, or room for muzzle attachments.  Re-barreling the thing or attaching a grenade launcher seem off too.

Accessories, sights, and stock can be lived with IMO.